mlos_bench/mlos_bench/launcher.py

```diff
@@ -142,6 +146,15 @@ def _parse_args(parser: argparse.ArgumentParser) -> Tuple[argparse.Namespace, Li
             help='Path to the storage configuration file.' +
                  ' If omitted, use the ephemeral in-memory SQL storage.')
 
+        parser.add_argument(
+            '--random_init', required=False, default=False,
+            dest='random_init', action='store_true',
+            help='Initialize tunables with random values. (Before applying --tunable_values).')
+
+        parser.add_argument(
+            '--random_seed', required=False, type=int,
```

Comment:
Should add this to the cli config schema.

Reply:
done